general-concepts/use-flags/text.xml

```diff
@@ -310,17 +310,17 @@ GnuTLS is more featureful than OpenSSL, it is favoured:
 
 <codesample lang="ebuild">
 src_configure() {
-	local myconf
+	local myconf=()
 
 	if use ssl; then
-		myconf+=" --enable-ssl --with-ssl=$(usex gnutls gnutls openssl)"
+		myconf+=( --enable-ssl --with-ssl="$(usex gnutls gnutls openssl" )
```

Comment:
```suggestion
		myconf+=( --enable-ssl --with-ssl="$(usex gnutls gnutls openssl)" )
```

Reply:
Huh, email replies on PRs don't go on the conversation... TIL.